src/storage/test/TestUtils.h

```diff
@@ -74,7 +74,31 @@ class TestUtils {
                                                             localhost,
                                                             workers);
         store->init();
-        sleep(1);
+
+        // When useMetaServer is true, we deal with it using waitUntilAllElected.
+        if (useMetaServer) {
+            sleep(1);
+        } else {
+            // Wait until all part leader exist, here spaceId can only be 0.
+            GraphSpaceID spaceId = 0;
+            while (true) {
+                int readyNum = 0;
+                for (auto partId = 0; partId < partitionNumber; partId++) {
+                    auto retLeader = store->partLeader(spaceId, partId);
+                    if (ok(retLeader)) {
+                        auto leader = value(std::move(retLeader));
+                        if (leader == localhost) {
+                            readyNum++;
+                        }
+                    }
+                }
+                if (readyNum == partitionNumber) {
+                    LOG(INFO) << "All leaders have been elected!";
+                    break;
+                }
+                usleep(100000);
+            }
+        }
```

Comment:
Here , Why not call waitUntilAllElected direct?

Reply:
when  useMetaServer is true,  use ` waitUntilAllElected(kvstore::KVStore* kvstore, GraphSpaceID spaceId, int partNum)`
when   useMetaServer  is flase, use ```
 waitUntilAllElected(kvstore::KVStore* kvstore, GraphSpaceID spaceId,
        std::set<PartitionID> parts)

The difference between these two functions is whether to use metaserver, so the start of partId is also different, one starts from 0 and the other starts from 1.

```